sigmf-spec.md

```diff
@@ -425,6 +428,11 @@ fields may be relative to baseband. It is required that both `freq_lower_edge`
 and `freq_upper_edge` be provided, or neither; the use of just one field is not
 allowed.
 
+The `label` field may be used for any prupose, but should be limited to not more
+than 20 charachters for user interface compatibility. Similarily, any user
+interface that makes use of this field should consider visibility of up to 20
+charachters.
```

Comment:
"characters"

When you change this perhaps mention the purpose of `label` is to be a short identifier for GUIs.

Reply:
Fixed (thanks - and I had a `prupose` in there too... wow). I tweaked the language a bit to make sure your point comes across.